lolex.js

```diff
@@ -200,9 +200,16 @@ function withGlobal(_global) {
         if (!clock.jobs) {
```

Comment:
remove the changes in this file?

Reply:
Ah - I thought it was ignored o_0

Reply:
yeah, no build artifacts, unless its a release commit. not totally sure how to ignore this happening again and again, seeing that one cannot ignore files already added to the index, but adding a `precommit` hook to remove changes could be reasonable.

Reply:
Could move from `prepublish` to `prepublishOnly`?

https://docs.npmjs.com/misc/scripts

Reply:
I'll send a PR